rir/src/compiler/opt/elide_env.h

```diff
@@ -1,6 +1,8 @@
 #ifndef PIR_ELIDE_ENV_H
 #define PIR_ELIDE_ENV_H
 
+#include "../translations/rir_compiler.h"
```

Comment:
that's the wrong include. `pir_translator.h` should be enough, not? (more instances of this below and above)